openupgradelib/openupgrade.py

```diff
@@ -2970,6 +2971,39 @@ def format_message(f):
             f.active = False
 
 
+def add_column(cr, field_name, table_name, sql_type, init_value=False):
```

Comment:
```suggestion
def add_column(cr, field_name, table_name, sql_type, init_value=False, translatable=False):
```

and switch the type for char accordingly.

Reply:
This would make sense if I was passing the field_type, but at the moment I am passing only the sql_type, then the person who call this method should put "jsonb" in sql_type directly instead of using this translatable flag.

Reply:
Then I insist this is not what we are looking for. We want the field_type, not the sql_type. Stefan agrees with me.

Reply:
And analogous to that, maybe the model name?

Reply:
@hbrunn but this `add_columns` should be called with `field_type` or `sql_type`? with `model_name` or `table_name`?

Reply:
fine with me